### DISCUSSION TITLE
Keyball61 LEDのパッドが剥離してしまいました

### DISCUSSION THREAD

**Discussion by OQFFF** [Yowkees/keyball#264] [Q&A] [answered]
![29_20230628170750](https://github.com/Yowkees/keyball/assets/104715416/af92dc16-99cd-45b6-8fab-04e383d42113)
トラックボール側、No4の左下(画像赤丸部分)のパッドが剥離してしまいました。
回路図を拝見したところ、GNDに接続すれば大丈夫だと思いますが、具体的にどこの部分に配線すればよいのか分かりません。
ご教示いただけますと幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
GNDではなくDinですね。
画像に移っていない３番のLEDとつなぎます。
はく離した個所のLEDの足と、下記画像の赤点をエナメル線等で接続してください。お手数ですがご確認お願いいたします。
![KB61R_LEDfix](https://github.com/Yowkees/keyball/assets/84209772/6a127b58-a997-417d-9394-5d19efaaede5)

**Comment by OQFFF** [comment]
回答ありがとうございます。
先程配線し、全てのLEDが正常に点灯しました☺️

**Comment by Yowkees** [comment]
修理対応ありがとうございました！
